src/ngraph/CMakeLists.txt

```diff
@@ -345,7 +345,7 @@ endif()
 
 if (APPLE)
     set_property(TARGET ngraph PROPERTY PREFIX "lib")
-    set_property(TARGET ngraph PROPERTY OUTPUT_NAME "ngraph.so")
+    set_property(TARGET ngraph PROPERTY OUTPUT_NAME "ngraph.dylib")
```

Comment:
Not sure what these three lines do, but it looks like they construct the output name. Do we have to use this? What happens if we don't use these three lines?
If you want to use these then let's use them what looks to be the correct way. Set the SUFFIX on line 349 to be ${CMAKE_SHARED_LIBRARY_SUFFIX}. 
I suspect we can remove lines 346 - 350